domain/src/test/java/org/oppia/android/domain/topic/StoryProgressTestHelperTest.kt

```diff
@@ -492,6 +499,116 @@ class StoryProgressTestHelperTest {
     assertThat(storySummary.chapterList[1].chapterPlayState).isEqualTo(ChapterPlayState.COMPLETED)
   }
 
+  @Test
+  fun testProgressTestHelper_markFullProgressForAllTopicsMoreThanOneWeek_ongoingTopicListEmpty() {
+    storyProgressTestHelper.markFullProgressForAllTopics(
+      profileId,
+      /* timestampOlderThanOneWeek= */ true
+    )
+    testCoroutineDispatchers.runCurrent()
+
+    topicController.getOngoingTopicList(
+      profileId
+    ).toLiveData().observeForever(mockOngoingTopicListObserver)
+    testCoroutineDispatchers.runCurrent()
+
+    verifyGetOngoingTopicListSucceeded()
+
+    val ongoingTopicList = ongoingTopicListResultCaptor.value.getOrThrow()
+    assertThat(ongoingTopicList.topicList).isEmpty()
+  }
+
+  @Test
+  fun testProgressTestHelper_markFullProgressForAllTopicsLessThanOneWeek_ongoingTopicListEmpty() {
+    storyProgressTestHelper.markFullProgressForAllTopics(
+      profileId,
+      /* timestampOlderThanOneWeek= */ false
```

Comment:
this can be a named argument, as you can also add named argument or profileId too, so that all the params have named arguments.

Reply:
Done